cvmfs/statistics.h

```diff
@@ -68,7 +68,7 @@ class Statistics {
   ~Statistics();
   Statistics *Fork();
   Counter *Register(const std::string &name, const std::string &desc);
-  Counter *Lookup(const std::string &name);
+  Counter *Lookup(const std::string &name) const;
```

Comment:
This still isn't completely "const safe", since the returned pointer can be used to modify the counter.
We need to stop here, since changing anything else would require a more complicated refactoring of the `Counter` class.